pages/linux/arch-chroot.md

```diff
@@ -0,0 +1,24 @@
+# arch-chroot
+
+> Enhanced `chroot` command to help in the Arch Linux installation process.
+> More information: <https://man.archlinux.org/man/arch-chroot.8>.
+
+- Start an interactive shell in a new root directory:
+
+`arch-chroot {{path/to/new/root}}`
+
+- Specify the user to run as:
+
+`arch-chroot -u {{user}} {{path/to/new/root}}`
+
+- Run a command in a new root directory:
+
+`arch-chroot {{path/to/new/root}} {{command}} {{command_arguments}}`
+
+- Generate the initial ramdisk using the preset provided by the `linux` package on new root directory:
+
+`arch-chroot {{path/to/new/root}} {{mkinitcpio --preset linux}}`
+
+- Start a zsh session in a new root directory (the `zsh` package should be installed in the target):
```

Comment:
How about moving this example to be next to the "custom user" one above? The sequencing of the page may make more sense that way IMO:

```suggestion
- Specify the shell, other than the default `bash` (in this case, the `zsh` package should have been installed in the target system):
```

(sorry if some of my comments don't make much sense, I'm a little out of my depth here.)